README.md

```diff
@@ -80,6 +80,7 @@ Game networking is a subset of computer networking that specifically covers data
 
 - [C / ENet](http://enet.bespin.org/) - Simple and robust reliable UDP networking library.
 - [C# / Barebones Master Server](https://github.com/alvyxaz/barebones-masterserver) - Backend framework with auth, profile, lobby, chat features for Unity.
+- [C# / ENet-CSharp](https://github.com/nxrighthere/ENet-CSharp) - Reliable UDP networking library extended for the .NET environment.
```

Comment:
We already have the original ENet linked above, one might easily find C#-ported project if wanted.